community/kernel/src/main/java/org/neo4j/kernel/impl/store/format/RecordFormats.java

```diff
@@ -58,5 +68,22 @@ public Factory( String key, String... altKeys )
 
     RecordFormat<RelationshipTypeTokenRecord> relationshipTypeToken();
 
-    RecordFormat<RelationshipGroupRecord> relationshipGroup();
+    RecordFormat<DynamicRecord> dynamic();
+
+    boolean hasStore( StoreType store );
```

Comment:
Oh, right that method came about before capabilities existed. I agree with this. Lemme make this change.